### PR TITLE
system-variables: add doc for tidb_expensive_txn_time_threshold

### PR DESCRIPTION
**Pull request by crazycs520**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14202
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -2389,6 +2389,24 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
     - Slow logs are printed after the statement is executed.
     - Expensive query logs print the statements that are being executed, with execution time exceeding the threshold value, and their related information.
 
+### tidb_expensive_txn_time_threshold
```
Comment (Oreoxmt) [inline system-variables.md:2392]:
```suggestion
### tidb_expensive_txn_time_threshold <span class="version-mark">New in v7.2.0</span>
```

File: system-variables.md [hunk 2]
```diff
@@ -2389,6 +2389,24 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+
+</CustomContent>
+
+- Scope: GLOBAL
+- Persists to cluster: No, only applicable to the current TiDB instance that you are connecting to.
+- Type: Integer
+- Default value: `600`
+- Range: `[60, 2147483647]`
+- Unit: Seconds
+- This variable is used to set the threshold value that determines whether to print expensive txn logs. Expensive txn logs print the related information of transactions that have not been committed or rollback and the duration of the transaction exceeds the threshold value.
```
Comment (Oreoxmt) [inline system-variables.md:2408]:
```suggestion
- This variable sets the threshold for logging expensive transactions. An expensive transaction is one that is neither committed nor rolled back, and the duration of the transaction exceeds this threshold.
```

File: system-variables.md [hunk 3]
```diff
@@ -2389,6 +2389,24 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+
+</CustomContent>
+
+- Scope: GLOBAL
+- Persists to cluster: No, only applicable to the current TiDB instance that you are connecting to.
+- Type: Integer
+- Default value: `600`
+- Range: `[60, 2147483647]`
+- Unit: Seconds
+- This variable sets the threshold for logging expensive transactions. An expensive transaction is one that is neither committed nor rolled back, and the duration of the transaction exceeds this threshold.
```
Comment (qiancai) [inline system-variables.md:2408]:
```suggestion
- This variable controls the threshold for logging expensive transactions, which is 600 seconds by default. When the duration of a transaction exceeds the threshold, and the transaction is neither committed nor rolled back, it is considered an expensive transaction and will be logged.
```

**Review by qiancai — APPROVED** [review]
/lgtm

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**